src/lib/create/from-string.js

```diff
@@ -63,12 +63,13 @@ export function configFromISO(config) {
         allowTime,
         dateFormat,
         timeFormat,
-        tzFormat;
+        tzFormat,
+        isoDateLen = isoDates.length,
+        isoTimeLen = isoTimes.length;
```

Comment:
Could you change these to `isoDatesLen` and `isoTimesLen`?

Reply:
These are just a variables for isoDates and isoTimes length which are used in 72 and 84th line

Reply:
@marwahaha I have updated these variables